service/src/main/kotlin/fi/espoo/evaka/shared/auth/Acl.kt

```diff
@@ -114,13 +114,9 @@ WHERE employee_id = :userId AND placement.id = :placementId
             it.createQuery(
                 // language=SQL
                 """
-SELECT acl.role
-FROM person ch
-LEFT JOIN placement pl ON pl.child_id = ch.id AND pl.end_date > current_date - INTERVAL '1 month'
-LEFT JOIN application a ON a.child_id = ch.id AND a.status = ANY ('{SENT,WAITING_PLACEMENT,WAITING_CONFIRMATION,WAITING_DECISION,WAITING_MAILING,WAITING_UNIT_CONFIRMATION, ACTIVE}'::application_status_type[])
-LEFT JOIN placement_plan pp ON pp.application_id = a.id
-JOIN daycare_acl acl ON acl.daycare_id = pl.unit_id OR acl.daycare_id = pp.unit_id
-WHERE employee_id = :userId AND ch.id = :childId
+SELECT role
+FROM child_acl_view
+WHERE employee_id = :userId AND child_id = :childId
```

Comment:
Since we introduce `R__child_acl_view.sql`, I thought we might want to use that here too.